### PR TITLE
Improve chrashlog

### PR DESCRIPTION
**Pull request by miguelinux**
Hi @chengangc ,

This is my start of the proposal, it is no finished, I count finish today, i will finish it tomorrow.

I'm send you this PR to you so we can review this proposal and then when you aproval send to the mailing list or directly to the upstream repo.

### CODE REVIEW

**Comment by chengangc** [comment]
Thanks a lot for your help, Miguel.

So in your following patch/es, there will be somewhere to handle the watchdog.conf file?

**Comment by miguelinux** [comment]
Hi @chengangc 

Just finished the patch set, please test it and let me know any concerns.

I rebased to the current ACRN master tree.

sorry for this delay

File: tools/acrn-crashlog/Makefile [hunk 1]
```diff
@@ -136,15 +138,18 @@ uninstall:
 	@if [ -e "$(DESTDIR)/usr/share/acrn/crashlog/40-watchdog.conf" ];then \
 		$(RM) $(DESTDIR)/usr/share/acrn/crashlog/40-watchdog.conf; \
 	fi
+	@if [ -e "$(DESTDIR)/usr/share/acrn/crashlog/80-coredump.conf" ];then \
```
Comment (chengangc) [inline tools/acrn-crashlog/Makefile:142]: Is it better to check whether there is something residual under /etc/sysctl.d and /etc/systemd/system.conf.d/?

File: tools/acrn-crashlog/data/usercrashctl [hunk 2]
```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
```
Comment (chengangc) [inline tools/acrn-crashlog/data/usercrashctl]: Can we rename this file as crashlogctl? Since usercrash is part of features of the crashlog.
Thanks.

**Review by chengangc — COMMENTED** [review]
Hi @miguelinux ,

Thanks a lot for your help. Please take a look on my comments. 

Thanks.

File: tools/acrn-crashlog/data/usercrashctl (hunk 2 above)
Comment (miguelinux) [reply tools/acrn-crashlog/data/usercrashctl]: Sure, will do

File: tools/acrn-crashlog/Makefile (hunk 1 above)
Comment (miguelinux) [reply tools/acrn-crashlog/Makefile:142]: Yes, but lets the crashlogctl do the work.
I'll work on that.

**Comment by miguelinux** [comment]
Hi @chengangc 

I did the modification, let me know your opinion.

File: tools/acrn-crashlog/Makefile [hunk 3]
```diff
@@ -102,24 +102,27 @@ install:
...
 .PHONY:uninstall
 uninstall:
 	@if [ -e "$(DESTDIR)/usr/bin/acrnprobe" ];then \
 		$(RM) $(DESTDIR)/usr/bin/acrnprobe; \
 	fi
-	@if [ -e "$(DESTDIR)/usr/bin/acrnprobe_prepare.sh" ];then \
-		$(RM) $(DESTDIR)/usr/bin/acrnprobe_prepare.sh; \
+	@if [ -e "$(DESTDIR)/usr/bin/crashlogctl" ];then \
+		$(DESTDIR)/usr/bin/crashlogctl disable && \
+		$(RM) $(DESTDIR)/usr/bin/crashlogctl; \
```
Comment (miguelinux) [inline tools/acrn-crashlog/Makefile:125]: Here I call `crashlogctl diable`to erase any installation made by user.

**Pull request merged**